nbdev/cli.py

```diff
@@ -82,13 +82,30 @@ def _render_nb(fn, cfg):
     fn.write_text(txt)
 
 # %% ../nbs/09_API/12_cli.ipynb 11
+def _update_repo_meta(cfg):
+    "Enable gh pages and update the homepage and description in your GitHub repo."
+    token=os.getenv('GITHUB_TOKEN')
+    if token: 
+        from ghapi.core import GhApi
+        api = GhApi(owner=cfg.user, repo=cfg.repo, token=token)
+        try: # repo needs something in it before you can enable pages
+            cmds = L(['git add settings.ini', "git commit -m'add settings'", 'git config push.default current', 'git push']) 
+            cmds.map(partial(run, ignore_ex=True))
+            api.enable_pages(branch='gh-pages')
```

Comment:
If the repo is empty, we get a `403 error: conflict` if trying to enable pages.  Depending on how you create your repo before initialiing nbdev this is problematic.  This is why the git commit step exists above.